### ISSUE TITLE
Use new audioplayers API

### ISSUE THREAD

**Issue by luanpotter** [flame-engine/flame#86]
The audio_pool class is using deprecated methods from audioplayers:

```
Fix lib/audio_pool.dart. (-1.49 points)

Analysis of lib/audio_pool.dart reported 3 hints:

line 50 col 13: 'completionHandler' is deprecated and shouldn't be used.

line 60 col 16: 'completionHandler' is deprecated and shouldn't be used.

line 62 col 16: 'completionHandler' is deprecated and shouldn't be used.
```

We need to fix that ASAP to use the new stream APIs.

**Comment by erickzanardo** [comment]
@luanpotter Do you think we still need the AudioPool class now that we have the LOW_LATENCY mode?

**Comment by luanpotter** [comment]
@erickzanardo not sure... maybe we could have it for larger audios? is low latency mode good with very quick, short firing audios that need to be played immediately?

**Comment by erickzanardo** [comment]
Yeah, but do you think we need audio pool for larger files, wouldn't that heavy on memory?

**Comment by luanpotter** [comment]
yes, for sure. but that's the trade off. Let's first replace audiopool with low_latency on all our games and if we have complete success, we can get rid of it.

**Comment by erickzanardo** [comment]
Resolved on 0.14.1
